app/Policies/CommentPolicy.php

```diff
@@ -47,19 +47,6 @@ public function viewAny(?User $user, Model $commentable): bool
     public function create(?User $user, ?Model $commentable = null, ?int $articleType = null): bool
     {
         if ($user?->isMuted()) {
-            // Even when muted, developers may still comment on tickets for their own achievements.
-            // TODO this is silly. delete all of this.
-            if ($commentable !== null && $commentable instanceof \App\Models\Ticket) {
-                $commentable->loadMissing(['achievement.developer']);
-
-                $didAuthorAchievement = $commentable->achievement->developer->id === $user->id;
-
-                return
-                    $didAuthorAchievement
-                    && $commentable->is_open
-                    && $user->hasAnyRole([Role::DEVELOPER_STAFF, Role::DEVELOPER]);
-            }
```

Comment:
Taking the opportunity now to remove all this stuff.

Reply:
Seems like it might still be necessary.
```
// Even when muted, developers may still comment on tickets for their own achievements.
```
After setting my `muted_until`, I'm unable to comment on tickets for my achievements.

Reply:
Then it's working as intended.

Discussion here: https://discord.com/channels/476211979464343552/1002688331827658774/1319427748216967242